library/sketches-library/src/main/java/uk/gov/gchq/gaffer/sketches/datasketches/sampling/serialisation/ReservoirItemsSketchSerialiser.java

```diff
@@ -62,4 +64,23 @@ public boolean preservesObjectOrdering() {
     public boolean isConsistent() {
         return false;
     }
+
+    @Override
+    public boolean equals(final Object o) {
+        boolean rtn = (this == o);
```

Comment:
Can we keep these to a standard form please, they are just easier to read if they all read the same, something like this:
``` @Override
    public boolean equals(final Object obj) {
        if (this == obj) {
            return true;
        }

        if (null == obj || getClass() != obj.getClass()) {
            return false;
        }

        final ReservoirItemsSketchSerialiser serialiser = (ReservoirItemsSketchSerialiser) obj;

        return new EqualsBuilder()
                .append(arrayOfItemsSerDe, serialiser.arrayOfItemsSerDe)
                .isEquals();
    }```